apps/collection/acceptance/pages/nominated-person.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+let I;
+
+module.exports = {
+
+  _init() {
+    I = require('so-acceptance/steps.js')();
```

Comment:
I don't know. I've never used so-acceptance before. But if that is the case, then what you suggest makes sense